src/agoradatatools/etl/utils.py

```diff
@@ -1,7 +1,10 @@
 import synapseclient
 import yaml
+import pandas as pd
+import numpy as np
 
 
+# TODO these utils functions are not protected... refactor removing "_"
```

Comment:
what do you mean by these utils functions are not protected?

Reply:
I was referring to the convention where functions that start with `_` are usually only used internally (in the same module) but since these are `utils` functions they are used outside of the `utils.py` module. "Protected" or not is not the correct term though so I will edit my comment to be more clear.